src/pages/Explore_Regions.py

```diff
@@ -41,18 +47,32 @@ def plot_deaths_season(m_route, df):
     st.write(fig)
 
 
-# Still in Development, i don't know how to show it
-def plot_deaths_month(m_route, df):
-    dft = df[df['Migration route'] == m_route]
-    fig = px.line(dft, x='date', y='Total Number of Dead and Missing',
-                  color='Reported Month', title=f"Deaths per Season in {m_route}")
+def plot_deaths_month(m_route,cause, df_wout_cause,df_wcause):
+    if len(cause) > 0:
+        dft = df_wcause[(df_wcause['Migration route'] == m_route) & (df_wcause['Cause of Death'].isin(
+        cause))].groupby(['Migration route','date'])[['Total Number of Dead and Missing', 'Minimum Estimated Number of Missing',
+                                                                          'Number of Females', 'Number of Males', 'Number of Children']].sum().reset_index()
+    else: 
+        dft = df_wout_cause[df_wout_cause['Migration route'] == m_route]
+    fig = px.line(dft, x='date', y=['Total Number of Dead and Missing','Number of Females', 'Number of Males', 'Number of Children'], title=f"Plot of deaths by month along the {m_route} route.")
```

Comment:
It looks like the "y" names are just showing up as "value" on the graph 
<img width="741" alt="Screen Shot 2022-10-20 at 19 06 19" src="https://user-images.githubusercontent.com/53492091/197001042-eb00c47b-0cc3-4df8-85c1-71b417a9b5f6.png">